rust/src/applayer.rs

```diff
@@ -24,6 +24,40 @@ use crate::filecontainer::FileContainer;
 use crate::applayer;
 use std::os::raw::{c_void,c_char,c_int};
 
+#[repr(C)]
+#[derive(Debug)]
+pub struct AppLayerTxData {
+    /// config: log flags
+    log_flags: u8,
+
+    logged: LoggerFlags,
```

Comment:
naming: this is a little confusing.. log_flags and LoggerFlags?

Reply:
Fixed